# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/HttpConnectionMgrChild.h
# Commit: 2502449c2e24
# Full Hash: 2502449c2e24993d495c886c717e7f9546cd9e17
# Author: Kershaw Chang <kershaw@mozilla.com>
# Date: 2020-08-05 21:44:54
# Regressor Bug: 1497249
# File Overlap Count: 4
# Description:
#   Bug 1497249 - P2: ipdl for nsIWebSocketConnection r=michal
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D30624
# ==============================================================================

diff -r 432c7e9dc19f -r 2502449c2e24 netwerk/protocol/http/HttpConnectionMgrChild.h
--- a/netwerk/protocol/http/HttpConnectionMgrChild.h	Mon Aug 03 10:03:08 2020 +0000
+++ b/netwerk/protocol/http/HttpConnectionMgrChild.h	Wed Aug 05 15:17:38 2020 +0000
@@ -41,6 +41,12 @@
       Maybe<SpeculativeConnectionOverriderArgs> aOverriderArgs, uint32_t aCaps,
       Maybe<PAltSvcTransactionChild*> aTrans);
 
+  already_AddRefed<PWebSocketConnectionChild> AllocPWebSocketConnectionChild(
+      PHttpTransactionChild* aTransWithStickyConn);
+  mozilla::ipc::IPCResult RecvPWebSocketConnectionConstructor(
+      PWebSocketConnectionChild* aActor,
+      PHttpTransactionChild* aTransWithStickyConn) override;
+
  private:
   virtual ~HttpConnectionMgrChild();
 